package-testing/node-sdk-relay/.eslintrc.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
```

Comment:
please consider using and/or updating https://github.com/Eppo-exp/sdk-test-data/blob/main/package-testing/eslint.config.EXAMPLE.mjs